pegasus/src/homepage.rb

```diff
@@ -105,9 +105,7 @@ def self.get_heroes
   end
 
   def self.get_actions(request)
-    code_break_takeover = promote_code_break(request)
-    code_bytes_takeover = promote_code_bytes(request)
-    hoc2020_ai_takeover = promote_hoc2020_ai(request)
+    hero_type = show_single_hero(request)
```

Comment:
This seems like a nice bit of cleanup :) Do we need to keep the `promote_` methods around?

Reply:
Oh...yes you do. Nvm!